services/registry/src/routes/schema.ts

```diff
@@ -12,7 +13,7 @@ const getFromSource = async (source: string, res: express.Response) => {
         const gqlObjects = await sources[source].getAgogosObjects();
         res.send(gqlObjects);
     } catch (error) {
-        console.warn(`Failed to get from source - ${source}`, {
+        logger.warn(`Failed to get from source - ${source}`, {
```

Comment:
The properties (error in this case) should be passed before the message to be added to the final log line as JSON properties. When added after the message it will be concatenated to the message instead of added as additional properties in the final log JSON entry.
Also, why not put in source as a property as well?